### PR TITLE
Fix white space in filters

### PR DESCRIPTION
**Pull request by KvnMrs**
Issue: https://github.com/blockframes/blockframes/issues/7494

### CODE REVIEW

File: libs/ui/src/lib/static-autocomplete/group/group.component.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
         <mat-checkbox color="primary" [indeterminate]="(value | getMode:groups) === 'indeterminate'"
           [checked]="(value | getMode:groups) === 'checked'" (change)="checkAll($event.checked)" test-id="checked-all">
         </mat-checkbox>
-        <input #inputEl class="mat-input-element" [placeholder]="placeholder" [formControl]="search" test-id="search-input-control" />
+        <input (keydown)="$event.code === 'Space' ? $event.stopPropagation() : '' " #inputEl class="mat-input-element" [placeholder]="placeholder" [formControl]="search" test-id="search-input-control" />
```
Comment (phpgeekfr) [inline libs/ui/src/lib/static-autocomplete/group/group.component.html:9]: remove extra space 

```suggestion
        <input (keydown)="$event.code === 'Space' ? $event.stopPropagation() : ''" #inputEl class="mat-input-element" [placeholder]="placeholder" [formControl]="search" test-id="search-input-control" />
```

**Review by phpgeekfr — COMMENTED** [review]
For me this is ok, this fixes the issue without any side effects.
we can still do `ctrl+a` to select everything (same behavior as space)

@GrandSchtroumpf what do you think ?

File: libs/ui/src/lib/static-autocomplete/group/group.component.html [hunk 2]
```diff
@@ -6,7 +6,7 @@
         <mat-checkbox color="primary" [indeterminate]="(value | getMode:groups) === 'indeterminate'"
           [checked]="(value | getMode:groups) === 'checked'" (change)="checkAll($event.checked)" test-id="checked-all">
         </mat-checkbox>
-        <input #inputEl class="mat-input-element" [placeholder]="placeholder" [formControl]="search" test-id="search-input-control" />
+        <input (keydown)="$event.code === 'Space' ? $event.stopPropagation() : ''" #inputEl class="mat-input-element" [placeholder]="placeholder" [formControl]="search" test-id="search-input-control" />
```
Comment (GrandSchtroumpf) [inline libs/ui/src/lib/static-autocomplete/group/group.component.html:9]: Did you check the code is the same on every browser ?

create a method on the component : 
```
(keydown)="cancelSpace($event)'
```

```typescript
cancelSpace(event: Event) {
  if (event.code === 'Space') event.stopPropagation();
}
```

Comment (KvnMrs) [reply libs/ui/src/lib/static-autocomplete/group/group.component.html]: I replaced 'code' by 'key' it look better on MDN. The code depends on the type of keyboard as opposed to the key

**Review by phpgeekfr — APPROVED** [review]

**Pull request merged**